dist/pinny.js

```diff
@@ -467,6 +470,26 @@
             return $(document.activeElement);
         },
 
+        _isWithin: function($element, selector) {
+            return ($element.closest('.' + selector).length) ? true : false;
```

Comment:
So to address the two questions above:

> do you mind if I re-write the function

In terms of reusability, that would result in a fairly narrow usage function; it would only operate on classes of elements. What if you wanted to check if the child has a `form` as a parent? Or the element doesn't have a class, but an ID? The function above wouldn't work.

Zepto/jQuery extensions should operate on selector syntax, so that any selector will work. This will provide the greatest flexibility when using in different situations.

> Also, would it be worth it to have a separate utility file for these types of custom Zepto/jQuery extensions? Or for now these functions should just go at the bottom of the file (before the initialization code)?

We already have a separate repository for this type of thing. It's called [selector-utils](https://github.com/mobify/selector-utils), and is where you would add this type of utility and its associated tests.